src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-cluster.cpp

```diff
@@ -180,8 +168,8 @@ WebRTCTransportProviderServer::UpsertResultEnum WebRTCTransportProviderServer::U
         result = UpsertResultEnum::kInserted;
     }
 
-    MatterReportingAttributeChangeCallback(AttributeAccessInterface::GetEndpointId().Value(), WebRTCTransportProvider::Id,
-                                           WebRTCTransportProvider::Attributes::CurrentSessions::Id);
+    // Use the correct attribute change reporting
```

Comment:
I am not sure I follow this comment: is this a TODO or should we drop it? Is the next line not the correct thing?

Reply:
legacy comments, removed